packages/augur-ui/src/modules/create-market/components/common.tsx

```diff
@@ -55,6 +55,7 @@ import {
   DISMISSABLE_NOTICE_BUTTON_TYPES,
 } from 'modules/reporting/common';
 import PreviewMarketTitle from 'modules/market/components/common/PreviewMarketTitle';
+import { StringNullableChain } from 'lodash';
```

Comment:
We decided a while ago not to use lodash in the UI. They have had mem issues in the past.

Reply:
don't know how that got there. it isnt used